src/proto/Cosmos.proto

```diff
@@ -166,6 +166,13 @@ message Message {
         repeated Amount amounts = 3;
     }
 
+    // mayachain/MsgSend
+    message MAYAChainSend {
```

Comment:
`MAYAChainSend` is still not needed:
https://github.com/trustwallet/wallet-core/pull/3328/#discussion_r1287004918

Reply:
Removed